### PR TITLE
Add VSCode tasks for common tasks

### PR DESCRIPTION
**Pull request by arthuro555**
This makes working on GDevelop easier and faster, and makes it more friendly for new contributors to work on.

### CODE REVIEW

**Comment by Bouh** [comment]
[I've made an old batch script for Windows user ^^
](http://pastebin.fr/72288)

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		"type": "npm",
+		"script": "test",
+		"path": "GDJS/",
+		"group": "test",
+		"problemMatcher": [],
+		"label": "Run GDJS tests",
+		"detail": "Run tests for GDJS"
+	},
+	{
+		"type": "npm",
```
Comment (4ian) [inline .vscode/tasks.json:66]: You have this twice ;)

**Comment by 4ian** [comment]
Nice! Looks useful :)

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		"script": "test",
+		"path": "GDJS/",
+		"group": {
+			"kind": "test",
+			"isDefault": true
+		},
+		"problemMatcher": [],
+		"label": "Run newIDE tests",
+		"detail": "Run tests for newIDE"
```
Comment (4ian) [inline .vscode/tasks.json:43]: Be consistent for periods at the end of the sentence (some have it, some don't) of the `detail` field.

File: .vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+		"label": "Build GDevelop.js",
+		"detail": "Builds GDCore for newIDE"
+	},
+	{
+		"type": "npm",
+		"script": "format",
+		"path": "newIDE/app/",
+		"problemMatcher": [],
+		"label": "Format newIDE",
+		"detail": "Formats via prettier the javascript in the newIDE/app directory."
```
Comment (4ian) [inline .vscode/tasks.json:31]: nitpicking: `Run auto-formatting (with Prettier) for the newIDE/app directory`

(the fact that we're formatting JavaScript is a detail, Prettier can be used to format other languages too)

**Review by 4ian — APPROVED** [review]

**Comment by blurymind** [comment]
Nice! Looks helpful 😀

**Pull request merged**